cilium-cli/cli/clustermesh.go

```
@@ -176,6 +178,46 @@ func newCmdClusterMeshDisconnectWithHelm() *cobra.Command {
 	return cmd
 }
 
+func newCmdClusterMeshPolicyDefaultClusterPrepare() *cobra.Command {
+	params := clustermesh.Parameters{
+		Writer: os.Stdout,
+	}
+	var namespace string
+	var allNamespaces bool
+
+	cmd := &cobra.Command{
+		Use:   "prepare-policy-default-local-cluster",
+		Short: "List policies that would be affected by changing policy-default-local-cluster on a cluster",
+		Long:  ``,
+		RunE: func(_ *cobra.Command, _ []string) error {
+			var err error
+			if namespace == "" {
+				if namespace, _, err = k8sClient.RESTClientGetter.ToRawKubeConfigLoader().Namespace(); err != nil {
+					fatalf("Unable to determine default namespace:  %s", err)
```

Could likely just fallback to `metav1.NamespaceDefault` instead of failing hard.